packages/wonder-blocks-modal/components/one-column.modal.test.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+import * as React from "react";
+import {shallow} from "enzyme";
+
+import {
+    MediaLayoutContext,
+    MEDIA_DEFAULT_SPEC,
+} from "@khanacademy/wonder-blocks-layout";
+import OneColumnModal, {
+    SmallOneColumnModal,
+    LargeOneColumnModal,
+} from "./one-column-modal.js";
+import {mount, unmountAll} from "../../../utils/testing/mount.js";
+
+describe("OneColumnModal", () => {
+    beforeEach(() => {
+        unmountAll();
+    });
+
+    it("should render SmallOneColumnModal on mobile", () => {
+        // Arrange
+        const wrapper = mount(
```

Comment:
This is the action so it probably should go under `// Act`. Sometimes there isn't anything to arrange, and that's ok. Just have an empty `// Arrange` section.

Reply:
In the case we're testing the rendering so having it under `// Act` makes sense.  I got in the habit from some of the previous tests putting the `mount` under arrange.